src/vivarium/framework/lookup/interpolation.py

```diff
@@ -283,9 +293,9 @@ class Order0Interp:
 
     def __init__(
         self,
-        data,
+        data: pd.DataFrame,
         continuous_parameters: _ParameterType,
-        value_columns: List[str],
+        value_columns: list[str] | tuple[str, ...],
```

Comment:
I think you should leave this typed as `list[str]` and convert inputs to a list in the Interpolation `__init__`